go.mod

```diff
@@ -90,6 +90,7 @@ require (
 	google.golang.org/api v0.50.0
 	gopkg.in/mgo.v2 v2.0.0-20190816093944-a6b53ec6cb22 // indirect
 	gopkg.in/yaml.v2 v2.4.0
+	gotest.tools/gotestsum v1.6.4 // indirect
```

Comment:
I dont think minikube's go.mod needs this, might wanna not put it here.

Reply:
Done. No longer included in go.mod.